packages/expo-audio/src/ExpoAudio.ts

```diff
@@ -19,6 +19,11 @@ export const PLAYBACK_STATUS_UPDATE = 'playbackStatusUpdate';
 export const AUDIO_SAMPLE_UPDATE = 'audioSampleUpdate';
 export const RECORDING_STATUS_UPDATE = 'recordingStatusUpdate';
 
+const replace = AudioModule.AudioPlayer.prototype.replace;
+AudioModule.AudioPlayer.prototype.replace = function (source: AudioSource) {
+  return replace.call(this, resolveSource(source));
+};
+
```

Comment:
I think we should add the same comment as here:
https://github.com/expo/expo/blob/f236adb1779016e854c61f3b59b04c1c56a6d384/packages/expo-video/src/VideoPlayer.tsx#L7
So that we remember that this breaks lazy loading of the module